swagger2.0/oic.r.mqtt.conf.swagger.json

```diff
@@ -0,0 +1,164 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MQTT configuration",
+    "version": "2021-02-10",
+    "license": {
+      "name": "OCF Data Model License",
+      "url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+      "x-copyright": "Copyright 2021 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/mqttconfResURI" : {
+      "get": {
+        "description": "The Resource through the MQTT server information can be set.\n",
```

Comment:
Small grammar nit, change text to:
The Resource through which the MQTT server information cam be set.

Reply:
fixed